src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/pybossa/PyBossaPlatform.java

```diff
@@ -134,6 +141,9 @@ public Boolean isCalibrationAllowed() {
      * @throws UnidentifiedWorkerException if the worker cannot be identified
      */
     private String identifyWorker(Map<String, String[]> param) throws UnidentifiedWorkerException {
+        if (param == null) {
+            return "";
```

Comment:
this should not return an empty string, but instead throw an UnidentifiedWorkerException